### ISSUE TITLE
Doesn't parse "records"

### ISSUE THREAD

**Issue by realh** [strycore/fakegir#28]
fakegir seems to miss out "record" items, which is what gi calls structs. structs may have methods. They're quite important in some libraries, so fakegir not supporting them is a big hindrance.

**Comment by kastixx** [comment]
Does PR #29 fix that issue?

**Comment by realh** [comment]
Yes, but only partially, because it doesn't support fields. I'm not sure how best to deal with them. If output classes had a constructor that set those fields with type annotations, would jedi etc be able to introspect them?

**Comment by kastixx** [comment]
Probably it won't. It's not possible to assign a docstring to class attribute, here's an explanation: https://stackoverflow.com/questions/3051241/how-to-document-class-attributes-in-python .

A possible work-around would be adding an empty property for each field:

```python
class Foo:
    @property
    def field_name(self):
        """Docstring for field_name"""
        pass
```

**Comment by realh** [comment]
That sounds like a good idea. Shall I give it a try and open a new PR?
